sky/shell/gpu/direct/ganesh_canvas.cc

```diff
@@ -51,7 +67,7 @@ SkCanvas* GaneshCanvas::GetCanvas(int32_t fbo, const SkISize& size) {
   GrBackendRenderTargetDesc desc;
   desc.fWidth = size.width();
   desc.fHeight = size.height();
-  desc.fConfig = kSkia8888_GrPixelConfig;
+  DCHECK(SelectPixelConfig(&desc.fConfig));
```

Comment:
`DCHECK` should probably be `CHECK`.  The argument to `DCHECK` isn't evaluated in release builds.